vividus-plugin-saucelabs/src/main/java/org/vividus/sauce/SauceLabsWebDriverFactory.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2019-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.sauce;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+import org.jbehave.core.model.Scenario;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.vividus.bdd.context.IBddRunContext;
+import org.vividus.bdd.model.MetaWrapper;
+import org.vividus.bdd.model.RunningStory;
+import org.vividus.proxy.IProxy;
+import org.vividus.selenium.IBrowserWindowSizeProvider;
+import org.vividus.selenium.IWebDriverFactory;
+import org.vividus.selenium.VividusWebDriver;
+import org.vividus.selenium.VividusWebDriverFactory;
+import org.vividus.selenium.manager.IWebDriverManager;
+import org.vividus.selenium.manager.IWebDriverManagerContext;
+
+public class SauceLabsWebDriverFactory extends VividusWebDriverFactory
+{
+    private static final String SAUCE_OPTION = "sauce:options";
+
+    private ISauceConnectManager sauceConnectManager;
+    private boolean sauceConnectEnabled;
+
+    public SauceLabsWebDriverFactory(IWebDriverFactory webDriverFactory, IBddRunContext bddRunContext,
+            IWebDriverManagerContext webDriverManagerContext, IProxy proxy,
+            IBrowserWindowSizeProvider browserWindowSizeProvider, IWebDriverManager webDriverManager)
+    {
+        super(webDriverFactory, bddRunContext, webDriverManagerContext, proxy, browserWindowSizeProvider,
+                webDriverManager);
+    }
+
+    @Override protected void configureProxy(DesiredCapabilities desiredCapabilities)
```

Comment:
why is it empty? and why should it be overridden ?

Reply:
because I don't need to configure proxy while sauce labs run. all that I needed - is configuring with sauce connect